lib/tasks/move_supplier.rake

```diff
@@ -0,0 +1,30 @@
+# TODO: Remove this once we've added all suppliers to the all moves in all environments
+namespace :move do
+  desc 'Populates a move with a supplier'
+  task supplier: :environment do
+    total = Move.count
+    total_updated = 0
+
+    locations_with_supplier = Location.joins(:suppliers)
+
+    moves_having_supplier = Move.where.not(supplier_id: nil).count
+    moves_missing_supplier = Move.where.not(from_location: locations_with_supplier).where(supplier_id: nil).count
```

Comment:
as above, since it is a number, it's might be better to provite add `_count` or `number_`

Reply:
Done